uobtheatre/site_messages/migrations/0005_auto_20251001_1946.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.2.25 on 2025-10-01 19:46
+
+import django_tiptap.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("site_messages", "0004_message_title"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="message",
+            name="dismissal_policy",
+            field=models.CharField(
+                choices=[
+                    ("DEFAULT", "Dismissable (Default)"),
+                    ("SINGLE", "Single-Session Only"),
+                    ("BANNED", "Prevented"),
+                ],
+                default="DEFAULT",
+                help_text="The policy for a message's dismissal. By default messages are dismissable, and this choice is stored in the cache until the event is over. Single-Session Only messages can be dismissed, but dismissal is not cached. Prevented messages cannot be dismissed by the user: if used for a modal, this will essentially shut down that function (or the whole site, for a sitewide modal).",
+                max_length=7,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="message",
+            name="display_location",
+            field=models.CharField(
+                choices=[
+                    ("BANNER", "Banner"),
+                    ("SITEWIDE_MODAL", "Sitewide Modal"),
+                    ("BOOKING_MODAL", "Booking Modal"),
+                    ("PRODUCTION_CREATION_MODAL", "Production Creation/Edit Modal"),
+                ],
+                default="BANNER",
+                help_text="The location on the site where the message should be displayed. Banner messages are displayed in a banner at the top of the page. Landing Modal messages are displayed in a modal on the landing page. Booking Modal messages are displayed in a modal on the booking page. Production Creation/Edit Modal messages are displayed in a modal on the production creation and editing page.",
```

Comment:
FWIW: The term `Landing Modal` is still used in the help text here, because `uobtheatre/site_messages/models.py` hasn't been updated everywhere to reflect that, even though they've now been renamed to `Sitewide Modal`s. We should probably do a quick find & replace job accordingly!